web_src/css/markup/content.css

```diff
@@ -338,7 +338,6 @@
 
 .markup .emoji {
   max-width: none;
```

Comment:
Is it useful?

Reply:
Likely not, but need to investigate why it was added.

Reply:
Tested, seems useless too

Reply:
Can't find any usable blame info, so removed it.